src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -642,26 +632,16 @@ export default class AirportModel {
         return this._runwayCollection.getRunwayRelationshipForRunwayNames(primaryRunwayName, comparatorRunwayName);
     }
 
-    // TODO: Implement changing winds, then bring this method back to life
     /**
      * @for AirportModel
      * @method updateRunway
      */
     updateRunway() {
```

Comment:
out of curiosity, where is this method called?  is it part of the update loop?

Reply:
I don't think it's actually called anywhere

Reply:
actually, I found it [here](https://github.com/openscope/openscope/pull/762/files#diff-db21fe1f5294d56680983f0c7dd43c24R467)